Makefile

```diff
@@ -238,6 +238,7 @@ setup-config-prompts:
 	 if [ "$$persist_sandbox" = "true" ]; then \
 		 read -p "Enter a password for the sandbox container: " ssh_password; \
 		 echo "ssh_password=\"$$ssh_password\"" >> $(CONFIG_FILE).tmp; \
+		 echo "persist_sandbox=$$persist_sandbox" >> $(CONFIG_FILE).tmp; \
```

Comment:
What do you think, can we also set `false` above, at the `$${persist_sandbox:-true};` line? Since the default is false, it feels like setup-config should set it as false...

Reply:
Maybe I miss your point, but shouldnt default be true?

Reply:
I mean, the default set in the application is false: https://github.com/OpenDevin/OpenDevin/blob/dea9b5c258ae9ebcc82447150fc1cb72b1ad96f6/opendevin/core/config.py#L183

It would be consistent if setup-config defaults to false, while allowing the user to set it to true themselves, isn't it?

Obviously this PR doesn't depend on that, it's just a thought.